lib/tasks/setting.rake

```diff
@@ -53,7 +53,7 @@ namespace :setting do
       next unless Settings::Definition.exists? setting_name
       next unless ENV.has_key? env_var_name
 
-      setting = Setting.find_by name: setting_name
+      setting = Setting.find_or_initialize_by(name: setting_name)
```

Comment:
In case of the user, this was called from https://github.com/opf/openproject/blob/dev/packaging/scripts/postinstall#L20 with `setting_name == :email_delivery_method`, but that setting was never saved previously (new installation). `setting` then becomes nil